### ISSUE TITLE
func_175378_g: updateActivePotionEffects -> updateDragSplitting

### ISSUE THREAD

**Issue by kashike** [ModCoderPack/MCPBot-Issues#186]
This has absolutely nothing to do with potion effects.
`func_175378_g`: `updateActivePotionEffects` -> `updateDragSplitting`

**Comment by maruohon** [comment]
In 1.8.9 there used to be a method func_146980_g in GuiContainer which was named updateDragSplitting.
Now in 1.9 (mapping snapshot_20160319) func_146980_g is gone, and instead there is func_175378_g in GuiContainer, which is mapped to the updateActivePotionEffects name.

So I now have a problem, since InventoryEffectRenderer#updateActivePotionEffects() method will actually override the what should be updateDragSplitting() method in GuiContainer if I add an AT to make func_175378_g ie. the old updateDragSplitting() public instead of private.

**Comment by matthewprenger** [comment]
Fixed:
`func_175378_g`: `updateActivePotionEffects` -> `updateDragSplitting`

@maruohon Not sure I understand your issue. Yes it's a private method, but MCP can't solve that. Either use an AT, or perhaps PR the AT change to Forge if you think it would be worth it for other modders.

**Comment by maruohon** [comment]
No my issue was that there was a method called updateActivePotionEffects()  (or func_175378_g) in both the GuiContainer and the InventoryEffectRenderer classes, but they are not the same method. The one in GuiContainer was/is private, so the one in InventoryEffectRenderer did not override it even with the conflicting naming, until I apply my AT making the one in GuiContainer public. So the real issue here was that why are both those methods called/mapped as func_175378_g now, even though they are different methods?

**Comment by matthewprenger** [comment]
Oh well thats been fixed now. func_175378_g is no longer mapped to updateActivePotionEffects

**Comment by maruohon** [comment]
Hmm, am I reading this wrong, or is this still broken...?

According to the bot, now it is named correctly in the GuiContainer class, but now also the DIFFERENT method (which should still be called updateActivePotionEffects) in InventoryEffectRenderer is called updateDragSplitting. As I said, they are not the same method ie. the one in InventoryEffectRenderer should not override the one in GuiContainer. The one in InventoryEffectRenderer should still be called updateActivePotionEffects.

Basically the naming of func_175378_g (which should only exist in InventoryEffectRenderer) as updateActivePotionEffects was correct and should not have been changed. However for some reason the updateDragSplitting method in GuiContainer in 1.9 got mapped to the same name func_175378_g, where as in 1.8.9 it was called func_146980_g, which is now gone.

**Comment by matthewprenger** [comment]
OK I see the issue. Something weird definitely happened with these methods. I've reverted the change for now. I'll let @bspkrs figure this one out.

**Comment by bspkrs** [comment]
The bot is fixed, I just have to fix and update the live SRG file archives.

**Comment by maruohon** [comment]
There is still a broken mapping related to this: the what should be func_175378_g() aka. updateActivePotionEffects() method is still named func_146980_g aka. updateDragSplitting() in GuiContainerCreative.
